tests/Integration/PageTest.php

```diff
@@ -141,6 +141,38 @@ public function testGetText()
         $this->assertStringContainsString('Verdana', $text);
     }
 
+    /**
+     * @see https://github.com/smalot/pdfparser/pull/457
+     */
+    public function testGetTextPullRequest457()
+    {
+        // Document with text.
+        $filename = $this->rootDir.'/samples/bugs/PullRequest457.pdf';
+        $parser = $this->getParserInstance();
+        $document = $parser->parseFile($filename);
+        $pages = $document->getPages();
+        $page = $pages[0];
+        $text = $page->getText();
+
+        $this->assertTrue(1000 < \strlen($text));
+        $this->assertStringContainsString('SUPER', $text);
+        $this->assertStringContainsString('VOORDEEL', $text);
+        $this->assertStringContainsString('KRANT', $text);
+        $this->assertStringContainsString('DINSDAG', $text);
+        $this->assertStringContainsString('Snelfilterkoffie', $text);
+        $this->assertStringContainsString('AardappelenZak', $text);
+        $this->assertStringContainsString('ALL', $text);
+
+        // Force garbage collection to not interfere with other tests.
```

Comment:
Can you explain what problem you solve here exactly? Do you run in memory-leaks?

In general test have to be isolated and must not interfere with each other.

Reply:
The unit tests are failing because `testRetainImageContentImpact()` from the [Parser integration tests](https://github.com/smalot/pdfparser/blob/master/tests/Integration/ParserTest.php) checks if memory usage is under a certain amount after executing its test. However it appears that PHP doesn't run garbage collection fast enough after our new test here (or something of that nature) causing the pdf parsed here to still be in memory during that test.

Reply:
Perhaps it would be better to call gc_collect_cycles() before testRetainImageContentImpact() instead of after this test. However I was unsure if editing other tests to make a new test work would be appreciated.

Reply:
Editing other tests is fine if you found a problem. Please propose a change which works in your opinion.

Reply:
It took me a bit of testing but I think I've finally found a good way to fix the issue.

testRetainImageContentImpact() now checks against a baseline memory it has at the beginning of the test. Instead of an arbitrary value.